src/test/mod.rs

```diff
@@ -390,6 +390,14 @@ fn self_tests() {
         path.push("main.rs");
         files.push(path);
     }
+    // for crates that need to be included but lies outside src
+    let external_crates = vec!["check_diff"];
+    for external_crate in external_crates {
+        let mut path = PathBuf::from(external_crate);
+        path.push("src");
+        path.push("main.rs");
+        files.push(path);
+    }
```

Comment:
Just wanted to note that if we add a `lib.rs` at some point then we'll need to remember to add that here too